examples/reference-entity-model/steps/ingestion/loadCustomersJSON.step.json

```diff
@@ -9,10 +9,10 @@
   "collections": [
     "loadCustomersJSON"
   ],
-  "inputFilePath": "testData/customers",
-  "outputURIReplacement": ".*testData,''",
+  "inputFilePath": "input/json",
```

Comment:
Can you delete the testData directory too, since that's unused now?

Reply:
sure